### ISSUE TITLE
Resolve warnings in logs related to large page data and missing sharp library

### ISSUE THREAD

**Issue by onedr0p** [sct/overseerr#3076]
### Description

Using the official docker image there's a couple items in the logs that (hopefully) look like they can be low hanging fruit to solve.

```
Warning: data for page "/" is 179 kB which exceeds the threshold of 128 kB, this amount of data can reduce performance.
See more info here: https://nextjs.org/docs/messages/large-page-data
```

```
Warning: For production Image Optimization with Next.js, the optional 'sharp' package is strongly recommended. Run 'yarn add sharp', and Next.js will use it automatically for Image Optimization.
```

This was brought up in https://github.com/sct/overseerr/discussions/502

### Version

1.30.1

### Steps to Reproduce

Deploy official Docker image and view the logs of the container

### Platform

desktop

### Device

MacBook Pro

### Operating System

MacOS

### Browser

Chrome

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by sct** [comment]
The log about page size we are likely going to just disable. It's caused a lot of people to think something is wrong when nothing is. I'm actually bothered that Nextjs is sending the log in production builds.

As for the sharp issue, we will be migrating away from using nextjs's image optimization so they will resolve itself.

**Comment by sct** [comment]
Now that #3056 is merged, the sharp error will go away in the next release. I will adjust the "large page data" rule before the next release as well.

**Comment by flammable** [comment]
Could I reopen this? I'm 1.32.5 with the official Docker image, and I'm still seeing the "large page data" entries in the log:

```
Warning: data for page "/login" is 195 kB which exceeds the threshold of 128 kB, this amount of data can reduce performance.
See more info here: https://nextjs.org/docs/messages/large-page-data
```

If there's another way to suppress these messages (assuming they're benign), I'm happy to do that, too. Thanks!

**Comment by jpartain89** [comment]
I know that not only do the logs still show this issue,but there is a real world issue that occurs in relation to this: After logging in, the program behaves erratically: it used to completely crash for about 5 to 10 mins, and then would continue on like nothing happened, but only on first login.

**Comment by TheCatLady** [comment]
> Could I reopen this? I'm 1.32.5 with the official Docker image, and I'm still seeing the "large page data" entries in the log:

The fix actually didn't make it into the 1.32.5 release. The relevant commit is d7b83d22cee3d20db564cc0564d42802b02327e3 and will be included in the next release.

**Comment by flammable** [comment]
@TheCatLady Thank you!
